### PR TITLE
MEN-4839 Monitor: describe the streaming logs pattern search.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: 09.Add-ons/50.Monitoring/docs.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ The service does normal logging which you can see with the `journalctl -u mender
 command.
 
 ### Examples
-Before the detailed description of the add-on configuration and structure
-in details, we present two quick-start examples.
+Before we describe the add-on configuration and structure
+in details, we present three quick-start examples.
```
Comment (estenberg) [inline 09.Add-ons/50.Monitoring/docs.md:54]: in "detail" (not details)

File: 09.Add-ons/50.Monitoring/docs.md [hunk 2]
```diff
@@ -73,6 +73,20 @@ and enable it by creating the following symbolic link:
 The daemon will automatically pick the files and start checking the log file.
 
 
+#### Has a device switched to accepted state?
+Assume you want to get an alert once your device has moved to `accepted` state.
+To this end you can use the log subsystem to look into the output of `journalctl -u mender-client`
```
Comment (estenberg) [inline 09.Add-ons/50.Monitoring/docs.md:78]: "look into" -> "follow"? Or does it check it periodically rather? In either case, clarify (and my hope is we follow :).

File: 09.Add-ons/50.Monitoring/docs.md [hunk 3]
```diff
@@ -159,6 +173,10 @@ where `SERVICE_NAME` is arbitrary name used in logging and for identification,
 `LOG_PATTERN` is the regular expression we are trying to match to each
 line of the logs, and `LOG_FILE` is the path to the log file.
 
+!!! There is a special variant for a value of the `LOG_FILE` variable, if it begins
```
Comment (estenberg) [inline 09.Add-ons/50.Monitoring/docs.md:176]: Would highlight this more, journalctl is widely used.

Add a separate section for this (#####) and call it something like Log stream monitoring, mention that this is the way to monitor output from journalctl.

**Review by estenberg — CHANGES_REQUESTED** [review]
Can we have a better, more realistic use case for stream monitoring using journalctl?
I have a hard time seeing why someone would like an alert once Mender gets accepted - that seems like normal operation.

Some Mender monitoring use cases that I think makes more sense:
* OTA Download "hangs": It has been downloading for more than 1 hour.
* Catch all Error messages from Mender

But there are likely other use cases that may make sense on the system or critical application level, e.g. the "Input device disconnected" use case mentioned a bit earlier. It should be in the journalctl -k output.

**Comment by lluiscampos** [comment]
@merlin-northern After #1527, you need to launch again this PR targetting `master`.

**Comment by merlin-northern** [comment]
> @merlin-northern After #1527, you need to launch again this PR targetting `master`.

yes, thank you for keeping track. but, you know, by editing the PR we could have just changed the target branch :)

**Comment by lluiscampos** [comment]
I didn't close it! This was GitHub magic :mage:

**Comment by merlin-northern** [comment]
> I didn't close it! This was GitHub magic 🧙

oh, sorry for blaming then, it says so above: `lluiscampos closed this 8 hours ago`
so before the target branch is deleted we should edit and re-target, right?

**Comment by lluiscampos** [comment]
> oh, sorry for blaming then, it says so above: `lluiscampos closed this 8 hours ago`

No problem, I also blamed once a colleague for the very same reason... I think this specific behavior of GitHub is tricky: it closes automatically the PRs targeting branches that are deleted but sets the author of the "close" to whoever deleted the branch :shrug: 

> so before the target branch is deleted we should edit and re-target, right?

Yes. Probably a rebase on master too.

**Pull request closed without merging**